07_StagingAndMultipleQueues/main.cpp

```diff
@@ -246,15 +246,15 @@ class StagingAndMultipleQueuesApp final : public application_templates::BasicMul
 					.binding = 0,
 					.type = nbl::asset::IDescriptor::E_TYPE::ET_SAMPLED_IMAGE,
 					.createFlags = IGPUDescriptorSetLayout::SBinding::E_CREATE_FLAGS::ECF_NONE,
-					.stageFlags = IGPUShader::E_SHADER_STAGE::ESS_COMPUTE,
+					.stageFlags = IShader::E_SHADER_STAGE::ESS_COMPUTE,
```

Comment:
generally speaking if you are replacing `IGPUShader::E_SHADER_STAGE` or `ICPUShader::E_SHADER_STAGE` best to replace with `hlsl::ShaderStage` instead of `IShader`